src/system/rtos_hooks.c

```diff
@@ -29,11 +29,15 @@ void FIQInterrupt() {
 // Replacement for DataAbortInterrupt
 void DataAbortInterrupt() {
 	taskDISABLE_INTERRUPTS();
+	taskENTER_CRITICAL();
```

Comment:
Please remove this, using `taskENTER_CRITICAL()` here prevents the data abort from printing to serial for mysterious reasons. Also remove it from `PrefetchAbortInterrupt()`.

Reply:
Fixed in https://github.com/purduesigbots/pros/pull/651/commits/b8f302634494d455f5c0c74918423112030dcb65.